lib/r10k/util/subprocess/runner/jruby.rb

```diff
@@ -14,8 +14,7 @@ def initialize(argv)
   end
 
   def run
-    spawn_opts = @cwd ? {:chdir => @cwd} : {}
-    stdout, stderr, status = Open3.capture3(*@argv, spawn_opts)
+    stdout, stderr, status = Open3.capture3(@argv.join(' '))
```

Comment:
https://github.com/jruby/jruby/issues/2298

Sounds like JRuby 9000 has a much better implementation, but we probably are going to be using a JGit backend in production anyway, the shellgit provider is mostly for testing.